src/Database.php

```diff
@@ -263,6 +264,26 @@ public function selectCollection($collectionName, array $options = [])
     }
 
     /**
+     * Select a GridFS bucket within this database.
+     *
+     * @see Bucket::__construct() for supported options
+     * @param string $bucketName Name of the bucket to select (defaults to 'fs')
+     * @param array  $options    Bucket constructor options
+     * @return Bucket
+     */
+    public function selectGridFSBucket($bucketName = 'fs', array $options = [])
```

Comment:
I used this method signature to be consistent with `selectCollection`. Since the bucket name is incorporated into the options array in the `Bucket` class, I think it's better to do it as you suggested.